.travis.yml

```diff
@@ -9,6 +9,14 @@ rvm:
   - 2.3.0
   - ruby-head
   - rbx-2
+  - jruby-9.0.5.0
+  - jruby-head
+
+jdk:
+  - oraclejdk8
+
+before_install:
+  - '[ "$JRUBY_OPTS" != "" ] && export JRUBY_OPTS="-Xcompat.version=2.0 --server -Xcompile.invokedynamic=false -Xcli.debug=true --debug"'
```

Comment:
I pushed a fix for these, thanks!

Let me know if you want me to squash the commits into one.